### PR TITLE
More AC and added DDM to F18C

### PR DESCRIPTION
**Pull request by Wamwod**
-Added DMS/DDM and applied it to the hornet (the hornet uses DDM for precise bombing)
-Added both F-14 Variants to the coordsType ac list 
-Added the AV-8B to the AC List with the A10_C2
-Added A-10C because legacy

### CODE REVIEW

File: Scripts/Hooks/scratchpad-hook.lua [hunk 1]
```diff
@@ -260,9 +260,11 @@ local function loadScratchpad()
     local function coordsType()
         local ac = DCS.getPlayerUnitType()
         if ac == "FA-18C_hornet" then
-            return "DMS", true
-        elseif ac == "A-10C_2" then
+            return "DMS/DDM", true
```
Comment (rkusa) [inline Scripts/Hooks/scratchpad-hook.lua:263]: I am pretty sure that the precisce bombing of the F18 uses DMS. You enter degrees, minutes, seconds and then decimal seconds.

Example:
```
N 13°35'12.52", E 144°55'54.11"
N 13°35.208', E 144°55.901'
55 P BR 76205 02979
168m, 551ft
```

Is entered as follows (taing lat as the example):
image
image

So I don't think that there is a change necessary for the F18, unless I am missing something?

**Review by rkusa — CHANGES_REQUESTED** [review]
Thanks a lot for the PR! I've added an inline comment regarding the F18 change, the other additions look good to me. 👍

**Review by rkusa — APPROVED** [review]

**Comment by rkusa** [comment]
FYI, I've added some options to further adjust how DDM coordinates are displayed. This inculdes four decimal places for the F18, and a zero padded 3 width wide longitude for the F16: https://github.com/rkusa/dcs-scratchpad/blob/main/Scripts/Hooks/scratchpad-hook.lua#L277-L292

**Comment by Wamwod** [comment]
Awesome this works even better 👍 I don't really code and was adding the F14 and changed the F-18 myself so I figured I'd make a PR to help others out since the F-14 wasn't added yet

**Pull request merged**